fuel-tx/src/tests/valid_cases/input.rs

```diff
@@ -648,7 +772,11 @@ fn transaction_with_duplicate_contract_inputs_is_invalid() {
         .add_output(o)
         .add_output(p)
         .finalize()
-        .check_without_signatures(Default::default(), &Default::default())
+        .check_without_signatures(
```

Comment:
This is a trait method. It gets used in the other impls of this method.

But it sounds like you want `chain_id` as part of `ConsensusParams` so I think it will get removed in that case anyway.